Symfony/CS/Fixer/PSR0/Psr0Fixer.php

```diff
@@ -161,6 +160,12 @@ public function supports(\SplFileInfo $file)
             return false;
         }
 
+        $tokens = token_get_all(sprintf('<?php %s', $filenameParts[0]));
```

Comment:
In tests you have list of 65 keywords. but there is 66 keywords. They are listed in method I referred to.
Create similar method to check if token is predefined constant.

Then, leave here that tokenizer call (just change it to `Tokens::fromCode` and in `if` below check `$tokens[1]->isKeyword()` and your new method `$tokens[1]->isPredefinedConstant()`